docs/installation.rst

```diff
@@ -45,7 +45,8 @@ create an **environment file**, and **build the Docker image**::
 
     git clone https://github.com/nexB/scancode.io.git && cd scancode.io
     make envfile
-    docker compose build
+    apt install docker-compose
```

Comment:
We are not supporting only Linux and only Debian-based distros. We have two ways to deal with this. We can use docker-compose 1.x which is a Python package or the docker-compose 2.x which is a single executable is Go.